### PR TITLE
Changes display method of category page and plugins

### PR DESCRIPTION
**Pull request by FinalAngel**
This PR enables the user to choose if a description is visible in plugins or the category list view:

**enabled:**
image

**disabled:**
image

### CODE REVIEW

**Comment by FinalAngel** [comment]
tests wil fail cause @mkoistinen needs to fix the app-hook-reload on django 1.6

File: aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html [hunk 1]
```diff
@@ -12,10 +12,20 @@ <h3 class="panel-title">{{ qgroup.grouper }}</h3>
...
-                    {% render_model question "title" %}
-                    {% if not list %}<span class="badge">{{ qgroup.grouper }}</span>{% endif %}
-                </a>
+                {% if view.config.app_data.config.show_description %}
+                    <div class="list-group-item">
+                        <a href="{{ question.get_absolute_url }}">
+                            {% render_model question "title" %}
+                            {% if not list %}<span class="badge">{{ qgroup.grouper }}</span>{% endif %}
+                        </a>
+                        {% render_model question "answer_text" %}
```
Comment (vxsx) [inline aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html]: shouldn't this also be clickable?
like in this example http://getbootstrap.com/components/#list-group-custom-content

**Comment by intellectronica** [comment]
:+1:

**Comment by vxsx** [comment]
:+1:

**Pull request merged**